src/Interpreters/ProcessList.cpp

```diff
@@ -513,7 +513,7 @@ void QueryStatus::throwProperExceptionIfNeeded(const UInt64 & max_execution_time
         {
             String additional_error_part;
             if (!elapsed_ns)
```

Comment:
Shouldn't we add `additional_error_part` if `elapsed != 0` ?
```suggestion
            if (elapsed_ns)
```

Reply:
Looks like yes...